config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'edamams#new'
+  get '/recipe/new', to: 'edamams#new'
+  get '/recipe/index', to: 'edamams#index', as: 'recipe_list'
+  get '/recipe/:uri', to: 'edamams#show', as: 'recipe_details'
```

Comment:
I'm not sure that `new` is a good name for this route/action. You're not really creating a new recipe, you're doing a search.

Also, could you use `resources` for the index and show pages?